sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -13,34 +13,50 @@ Azure Cognitive Services Text Analytics is a cloud service that provides advance
 
 ### Prerequisites
 * An [Azure subscription][azure_sub].
-* An existing [Cognitive Services][cognitive_resource] or Text Analytics resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
+* An existing Cognitive Services or Text Analytics resource.
 
-If you use the Azure CLI, replace `<your-resource-group-name>`, `<your-resource-name>`, `<location>`, and `<sku>` with your values:
+### Create a Cognitive Services or Text Analytics resource
+Text Analytics supports both [multi-service and single-service access][cognitive_resource_portal]. Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Text Analytics access only, create a Text Analytics resource.
+
+You can create either resource using the [Azure Portal][cognitive_resource_portal] or [Azure CLI][cognitive_resource_cli]. Below is an example of how you can create a Text Analytics resource using the CLI:
+
+```PowerShell
+# Create a new resource group to hold the text analytics resource -
+# if using an existing resource group, skip this step
+az group create --name <your-resource-name> --location <location>
+```
 
 ```PowerShell
-az cognitiveservices account create --kind TextAnalytics --resource-group <your-resource-group-name> --name <your-resource-name> --location <location> --sku <sku>
+# Create text analytics
+az cognitiveservices account create \
+    --name <your-resource-name> \
+    --resource-group <your-resource-group-name> \
+    --kind TextAnalytics \
+    --sku <sku> \
+    --location <location> \
+    --yes
 ```
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
```

Comment:
Does it duplicate what [Azure CLI][cognitive_resource_cli] does at above, https://github.com/Azure/azure-sdk-for-net/pull/9887/files#diff-c183227058ebd27d819132b62a437f6eR21

Reply:
It is the same link, but I wanted to make it obvious that there is information on it about how to get location and sku